tests/apps/tethysapp-test_app/test.sh

```diff
@@ -0,0 +1 @@
+echo "test"
```

Comment:
Is this file necessary?

Reply:
We check that it is printed in a test. But we could take it out